### PR TITLE
Added IANA considerations for obsoleted algorithms. #16

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- specifies MD5 and SHA1 collisions
- add `Status` to IANA Digest Algorithms
- Set `obsoleted` status on MD5 and SHA1

### CODE REVIEW

File: draft-polli-resource-digests-http.md [hunk 1]
```diff
@@ -600,16 +638,45 @@ Response:
 
 # Security Considerations
 
+## Broken cryptographic algorithms
+
+Cryptographic algorithms are supposed to provide a proof of integrity
+usable eg. in signatures.
+
+To discourage mistakes, broken-cryptographic algorithms
+like `MD5` and `SHA-1` are now NOT RECOMMENDED.
```
Comment (LPardue) [inline draft-polli-resource-digests-http.md]: I would mofidy these two paragraphs a little and say
"Cryptogrphic alorithms are intended to provide a proof of integrity suited towards cryptographic constructions such as signatures. However, these rely on collision-resistance for their security proofs [CMU-836068]. The `MD5` and `SHA-1` algorithms are vulnerable to collisions attacks and they are NOT RECOMMENDED.

File: draft-polli-resource-digests-http.md [hunk 2]
```diff
@@ -600,16 +638,45 @@ Response:
...
-It's important to note that, being the Digest header an hash of a resource representation,
+It's important to note that, being the `Digest` header an hash of a resource representation,
 signing only the `Digest` header, without all the representation metatada (eg.
 the values of `Content-Type` and `Content-Encoding`) may expose the communication
 to tampering.
 
+`Digest` SHOULD always be used over a connection which provides
+integrity at transport layer that protects HTTP headers.
+
+A `Digest` header using NOT RECOMMENDED {digest-algorithms} MUST NOT be used in signatures.
```
Comment (LPardue) [inline draft-polli-resource-digests-http.md]: What does an endpoint do if it receives such a header. Reject the message and issue an error? This is usually written like "An endpoint that receives such a header SHOULD ..."

**Review by LPardue — CHANGES_REQUESTED** [review]
Some change suggestions but ignore if you disagree.

File: draft-polli-resource-digests-http.md (hunk 2 above)
Comment (ioggstream) [reply draft-polli-resource-digests-http.md]: maybe remove this statement or lowercasing the MUST NOT as a shouldn't?

**Pull request merged**